src/App.js

```diff
@@ -1,16 +1,15 @@
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { getRockets } from './redux/rockets/rocketsSlice';
```

Comment:
Hi @mrkamin, please, notice that you are deleting getRockets from import.

Try adding both of them: `import { getRockets, getMissions }` from ...